test/scripts/e2e_subs/tealprogs/abi-min_balance.json

```diff
@@ -0,0 +1,30 @@
+{
+    "name": "opCodes",
+    "appId": null,
```

Comment:
nit:  😅 we are changing this field `"appId"` to `network` in the new ABI spec: https://github.com/algorandfoundation/ARCs/pull/53, it seems totally optional to have this field or not, might be okay to remove that

Reply:
👍